scripts/update_model_yaml.py

```diff
@@ -0,0 +1,92 @@
+import argparse
```

Comment:
Should use the same name for update model? I see we have `update-model-yml.yml` and `update_model_yaml.py`